packages/contracts/src/dollar/core/StakingShare.sol

```diff
@@ -237,10 +214,6 @@ contract StakingShare is
         _setURI(tokenId, tokenUri);
     }
 
-    function setUri(string memory tokenUri) external onlyMinter {
```

Comment:
Why was this method removed? I think we should keep it in order to set URI for `ERC1155` contract

Reply:
Already implemented in ERC1155Ubiquity